README.md

```diff
@@ -170,7 +170,8 @@ Currently, the following package managers are supported:
 * [Bower](http://bower.io/) (JavaScript)
 * [Bundler](http://bundler.io/) (Ruby)
 * [Cargo](https://doc.rust-lang.org/cargo/) (Rust)
-* [Conan](https://conan.io/) (C / C++)
+* [Conan](https://conan.io/) (C / C++, *experimental* as the VCS locations often times do not contain the actual source
+  code, see #2037.)
```

Comment:
Nit: We do not use a trailing dot in the other comments.

Reply:
Done.